articles/applications/application-settings/machine-to-machine/index.md

```diff
@@ -0,0 +1,34 @@
+---
+description: Application settings for Machine to Machine Applications
+toc: true
+---
+
+# Application Settings: Machine to Machine Application
```

Comment:
Up until now you were using `Machine-to-Machine` and now I see `Machine to Machine`. We should use one term and preferably this should be aligned with the dashboard as well (which as I see, uses `Machine to Machine`)